client/my-sites/comments/comment/style.scss

```diff
@@ -678,3 +678,12 @@
 		display: none;
 	}
 }
+
+// Nested comment at max depth
+
+.card.comment {
+	&.is-at-max-depth {
+		margin-bottom: 0px;
+		box-shadow: 0 -1px transparentize($gray-lighten-20, 0.5);
```

Comment:
We should add spaces before/after parenthesis to comply with coding guidelines.

Reply:
Updated, I was under the impression that the prettier pre-commit hook would take care of that.

Reply:
Oh it seems that it is, just in a wrong way. :)